cmd/node/node.go

```diff
@@ -348,6 +352,25 @@ func (app SpacemeshApp) stopServices() {
 
 }
 
+func getEdIdentity() *signing.EdSigner {
+	// read key from file if exist
+	log.Info("Reading identity from file")
+	dataDir, err := filesystem.GetSpacemeshDataDirectoryPath()
+	if err != nil {
+		log.Error("Could not get data path err=%v", err)
+		return signing.NewEdSigner()
```

Comment:
You shouldn't hide the error. In such case you should report back the error (change the method signature to return also an error)

Reply:
Fixed